Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,8 +489,40 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    updateAntialiasing(): void {
+      if (this._object.getAntialiasingQuality() != 'Off') {
+        const antialiasingFilter =
+          this._antialiasingFilter ||
+          (this._antialiasingFilter = new PIXI.filters.FXAAFilter());
+        antialiasingFilter.enabled = true;
+        antialiasingFilter.multisample =
+          PIXI.MSAA_QUALITY[
+            this._object.getAntialiasingQuality().toUpperCase()
+          ] || PIXI.MSAA_QUALITY.LOW;
```

Comment:
This is a bit of a code smell. We should rather have an enum map the "GDevelop antialiasing types" to the "PIXI ones".

Higher above add 
```ts
enum GDevelopToPIXIAntialiasing {
  None = PIXI.MSAA_QUALITY.NONE
  Low = PIXI.MSAA_QUALITY.LOW
  Medium = PIXI.MSAA_QUALITY.MEDIUM
  High = PIXI.MSAA_QUALITY.HIGH
}
```

Then replace this with 
```suggestion
          PIXI.MSAA_QUALITY[GDevelopToPIXIAntialiasing[this._object.getAntialiasingQuality()]] 
          ?? PIXI.MSAA_QUALITY.MEDIUM;
```

Reply:
👍